cli/src/main/resources/help/module.txt

```diff
@@ -3,7 +3,7 @@ SYNOPSIS
   To add a module locally
 
     module add [--module-root-dir=modules_root_directory_path] --name=module_name 
-               [--slot=slot_name] --(absolute-)resources=list_of_jars [--resource-delimiter=delimiter]
+               [--slot=slot_name] [--resources=list_of_jars] [--absolute-resources=list_of_jars] [--resource-delimiter=delimiter]
```

Comment:
The content looks good to me. But purely for formatting reasons, please, don't make lines longer than 80 characters. It won't break the CLI but just to be consistent with the other help files. Thanks!